docs/en/01_Tutorials/02_Extending_A_Basic_Site.md

```diff
@@ -462,16 +462,19 @@ Nothing here should be new. The *StaffPage* page type is more interesting though
 	:::php
 	<?php
 	class StaffPage extends Page {
-		private static $db = array(
-		);
+
 		private static $has_one = array(
-			'Photo' => 'Image'
+			'Photo' => 'File'
```

Comment:
This is the `hard` problem to solve for this pull request; The problem is, if the type is NOT image, and an image is uploaded, do we provide the same support for that file as though it was saved in a DBFile('image')? In which case, what's the benefit of the distinction? Is it simply a type hint, or is it only a validation rule to limit allowed mime types?